core/server/master/src/main/java/alluxio/master/metastore/rocks/RocksBlockMetaStore.java

```diff
@@ -157,7 +157,7 @@ && new String(columns.get(2).getName()).equals(BLOCK_LOCATIONS_COLUMN),
       }
     }
     mRocksStore = new RocksStore(ROCKS_STORE_NAME, dbPath, backupPath, opts, columns,
-        Arrays.asList(mBlockMetaColumn, mBlockLocationsColumn));
+        Arrays.asList(mBlockMetaColumn, mBlockLocationsColumn), true);
```

Comment:
is it possible to move these rocksdb utilities to common? then we don't have make worker depends on master.

Reply:
See comment below.

Reply:
Fixed.